project/src/components/offer-details/offer-details.tsx

```diff
@@ -60,17 +61,36 @@ function OfferScreen(): JSX.Element {
     [dispatch],
   );
 
-
   const { id }  = useParams<{id: string}>();
 
+  const offerData = currentOffer as Offer;
+  const offersNear = nearbyOffers as Offer[];
+  const offersForMap = useMemo(
```

Comment:
```suggestion
  const mapOffers = useMemo(
```
массивы должны быть во множественном числе